index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav class="menu-bar">
+      <a class="name" href="#">Eyob yoseph</a>
+      <img class="humburger" src="img/Icon - Menu.png" alt="">
+    </nav>
+  </header>
+  <main>
+    <section class="intro-page">
+     <h1 class="greeting"> Hey There.I’m Eyob</h1>
+     <h3 class="job">I’am a Software Developer</h3>
```

Comment:
Kindly reduce the font-size of this `h1` tag so that the "Hey There" can be on the same line as what we have in the Figma design. 😊 📗 

| Your Design | Figma Design |
| :---:   | :---: | 
| <img width="349" alt="image" src="https://user-images.githubusercontent.com/73854999/219871141-106c0b0e-6b96-41b5-a935-2e327c981173.png">| <img width="349" alt="image" src="https://user-images.githubusercontent.com/73854999/219871114-6bfcb72e-9a40-454e-9cb6-e782346f5f5a.png">|

Reply:
I have corrected it according to Figma design